frontend/src/scenes/insights/InsightDisplayConfig.tsx

```diff
@@ -103,63 +116,89 @@ export function InsightDisplayConfig({ filters, activeView, disableTable }: Insi
                                 </>
                             )}
                         />
-                    </span>
+                    </ConfigFilter>
                 )}
 
                 {showIntervalFilter(activeView, filters) && (
-                    <span className="filter">
-                        <span className="head-title-item">
+                    <ConfigFilter>
+                        <span>
                             <span className="hide-lte-md">grouped </span>by
                         </span>
                         <IntervalFilter view={activeView} />
-                    </span>
+                    </ConfigFilter>
                 )}
 
                 {activeView === InsightType.TRENDS &&
                 !filters.breakdown_type &&
                 !filters.compare &&
                 (!filters.display || filters.display === ChartDisplayType.ActionsLineGraph) &&
                 featureFlags[FEATURE_FLAGS.SMOOTHING_INTERVAL] ? (
-                    <SmoothingFilter />
+                    <ConfigFilter>
+                        <SmoothingFilter />
+                    </ConfigFilter>
                 ) : null}
 
                 {activeView === InsightType.RETENTION && (
-                    <>
+                    <ConfigFilter>
                         <RetentionDatePicker />
                         <RetentionReferencePicker />
-                    </>
+                    </ConfigFilter>
                 )}
 
                 {showPathOptions && (
-                    <span className="filter">
+                    <ConfigFilter>
                         <PathStepPicker />
-                    </span>
+                    </ConfigFilter>
                 )}
 
                 {showComparePrevious[activeView] && (
-                    <span className="filter">
+                    <ConfigFilter>
                         <CompareFilter />
-                    </span>
+                    </ConfigFilter>
                 )}
             </div>
-            <div className="display-config-inner-row">
+            <div className="flex items-center space-x-2 flex-wrap my-2">
+                {activeView === InsightType.TRENDS && (
+                    <ConfigFilter>
+                        <span>Aggregation Axis format</span>
```

Comment:
I still find this name not self-explanatory, could we ask @joethreepwood for some feedback, or add a tooltip?